examples/gce-machinedeployment.yaml

```
@@ -49,6 +49,10 @@ spec:
             machineType: "n1-standard-2"
             # In GB
             diskSize: 50
+            # ether of two network/subnetwork could be specified
```

Not yet clear here about the options and how I would specify e.g. only subnetwork. How would I have to specify it? Expect a convention like with `global/networks/default` for the network.

---

based on https://www.terraform.io/docs/providers/google/r/compute_instance.html#network

>network - (Optional) The name or self_link of the network to attach this interface to. Either network or subnetwork must be provided.

>subnetwork - (Optional) The name or self_link of the subnetwork to attach this interface to. The subnetwork must exist in the same region this instance will be created in. Either network or subnetwork must be provided.

so it's OK to have ether of them

---

Probably double entry, not sure if browser got suggestion. Would write

```suggestion
            # Name or self_link of the network and subnetwork to attach this interface to; either
            # of two network and subnetwork can be provided; if both are empty default network
            # will be taken
```